samples/snippets/create_lite_reservation_example.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google Inc. All Rights Reserved.
```

Comment:
This header should just be:

`Copyright 2022 Google LLC`. See go/releasing/preparing#Apache-header

Please update here and in the other new files created in this PR.

Reply:
I updated the license headers for new files created by this PR. I can change the rest in a separate PR. Thanks for spotting this one.